src/hooks/useContainer.ts

```diff
@@ -105,9 +105,9 @@ export function useContainer() {
 				else {
 					let credentialImageURL = credentialHeader?.vctm?.display && credentialHeader.vctm.display[0] && credentialHeader.vctm.display[0][defaultLocale] ?
 						credentialHeader.vctm.display[0][defaultLocale]?.rendering?.simple?.logo?.uri
-						: null;
+						: 'https://picsum.photos/300/200';
 
-					if (!credentialImageURL) { // prrovide fallback method through the OpenID credential issuer metadata
+					if (!credentialImageURL) { // provide fallback method through the OpenID credential issuer metadata
```

Comment:
This fallback breaks if the issuer is not pre-configured in the database of the wallet-backend-server.